### PR TITLE
x11-terms/xfce4-terminal: fix build againt xfce-base/libxfce4ui-4.19

### PR DESCRIPTION
**Pull request by xdch47**
Closes: https://bugs.gentoo.org/904500

### CODE REVIEW

**Comment by xdch47** [comment]
--> rename / spell fixing in libxfce4ui breaks linking xfce_gtk_menu_append_sep{e,a}rator

File: x11-terms/xfce4-terminal/xfce4-terminal-1.0.4-r1.ebuild [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+BDEPEND="
+	dev-libs/libxml2
+	dev-util/intltool
+	sys-devel/gettext
+	virtual/pkgconfig
+"
+
+src_prepare() {
+	subst_args=(xfce_gtk_menu_append_sep{e,a}rator)
+	sed -E -i "s#${subst_args[0]}#${subst_args[1]}#" terminal/terminal-window{,-dropdown}.c || die
```
Comment (thesamesam) [inline x11-terms/xfce4-terminal/xfce4-terminal-1.0.4-r1.ebuild:38]: This is the kind of thing which is _really_ better for both:
1. a patch (... we don't want to apply it anymore once it's fixed, and it's a precise change);
2. a patch which then goes upstream.

Comment (xdch47) [reply x11-terms/xfce4-terminal/xfce4-terminal-1.0.4-r1.ebuild:38]: already upstream: https://github.com/xfce-mirror/xfce4-terminal/commit/d9e909946951fff0c216e3338df5adfbf9b2aa1f
i think this is just a like a "transition" ebuild fix.
we could also just wait…

Comment (xdch47) [reply x11-terms/xfce4-terminal/xfce4-terminal-1.0.4-r1.ebuild:38]: or introduce backword compatiblity in libxfce4ui by reverting/reverse applying
--> https://gitlab.xfce.org/xfce/libxfce4ui/-/merge_requests/88/diffs?commit_id=08076f487296a7a9673ad7e8547a48ba41beb74f

**Comment by mgorny** [comment]
Closing as obsolete, we're at 1.1.3 right now. Sorry for not looking into it earlier.

**Pull request closed without merging**